### ISSUE TITLE
Tab is ignored on first line

### ISSUE THREAD

**Issue by mageddo** [inacho/bootstrap-markdown-editor#11]
If a put a text with tab on first line(like a code) this tab is ignored on editor , the bug [can be reproduced here](http://jsfiddle.net/deFreitas/t476p7wp/1/)

```
<script>
$('#myEditor').markdownEditor({
  // Options
});
</script>
<div id="myEditor">
    System.out.println();
</div>
```

**Comment by mageddo** [comment]
Happen with space too

**Comment by mageddo** [comment]
@inacho Any chance of merge it?

**Comment by inacho** [comment]
The spaces issue is fixed in the latest release (v1.3.0) you can test it here:
http://jsfiddle.net/t476p7wp/2/
